astropy/coordinates/funcs.py

```diff
@@ -153,11 +154,14 @@ def get_sun(time):
     # We have to manually do aberration because we're outputting directly into
     # GCRS
     earth_p = earth_pv_helio[..., 0, :]
-    earth_v = earth_pv_helio[..., 1, :]
+    earth_v = earth_pv_bary[..., 1, :]
+
+    # convert barycentric velocity to units of c
+    earth_v = (earth_v*u.au/u.d).to(u.m/u.s)/c
```

Comment:
I normally try to use quantities throughout, but since we're connecting with erfa here, Ithink the following might be clearer:
```
# convert barycentric velocity to units of c, but keep as array for passing in to erfa
earth_v /= c.to(u.au/u.d).value
```